numpy/core/src/multiarray/einsum_p.h

```diff
@@ -0,0 +1,48 @@
+#ifndef _NPY_EINSUM_P_H_
+#define _NPY_EINSUM_P_H_
+
+#define PY_SSIZE_T_CLEAN
+#include "Python.h"
+#include "structmember.h"
+
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+#define _MULTIARRAYMODULE
+#include <numpy/npy_common.h>
+#include <numpy/arrayobject.h>
+#include <numpy/halffloat.h>
+#include <npy_pycompat.h>
+
+#include <ctype.h>
+
+#include "simd/simd.h"
+#include "convert.h"
+#include "common.h"
+#include "ctors.h"
+
+#define EINSUM_IS_ALIGNED(x) ((((npy_intp)x)&0xf) == 0)
```

Comment:
```suggestion
#define EINSUM_IS_ALIGNED(x) npy_is_aligned(x, NPY_SIMD_WIDTH)
```
That one can easily causes a crush.